boards/wemos-zero/board.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C)  2016 Freie Universität Berlin
+ *                2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_wemos-zero
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the WeMos Zero board
+ *
+ * @author      Hauke Pertersen  <hauke.pertersen@fu-berlin.de>
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
```

Comment:
Better put yourself here so no innocents are bugged with questions regarding the board :wink: